core/editor/src/input/input_mapper.rs

```diff
@@ -235,6 +235,10 @@ impl Default for Mapping {
 			entry! {action=DocumentMessage::NudgeSelectedLayers(NUDGE_AMOUNT, -NUDGE_AMOUNT), key_down=KeyArrowRight, modifiers=[KeyArrowUp]},
 			entry! {action=DocumentMessage::NudgeSelectedLayers(NUDGE_AMOUNT, NUDGE_AMOUNT), key_down=KeyArrowRight, modifiers=[KeyArrowDown]},
 			entry! {action=DocumentMessage::NudgeSelectedLayers(NUDGE_AMOUNT, 0.), key_down=KeyArrowRight},
+			entry! {action=DocumentMessage::ReorderSelectedLayer(-1), key_down=KeyLeftBracket, modifiers=[KeyControl]},
+			entry! {action=DocumentMessage::ReorderSelectedLayer(1), key_down=KeyRightBracket, modifiers=[KeyControl]},
+			entry! {action=DocumentMessage::ReorderSelectedLayer(i32::MIN), key_down=KeyLeftCurlyBracket, modifiers=[KeyControl]},
+			entry! {action=DocumentMessage::ReorderSelectedLayer(i32::MAX), key_down=KeyRightCurlyBracket, modifiers=[KeyControl]},
```

Comment:
These also need the <kbd>Shift</kbd> modifier key.

Reply:
This was discussed [above](https://github.com/GraphiteEditor/Graphite/pull/271#discussion_r670810447), I could only get this to work by listening for the curly bracket key, rather than shift + square bracket. Once #124 is resolved this can perhaps be changed. Should I leave a comment saying so?

Reply:
Ah, gotcha. Yes, please definitely leave a comment explaining the situation to anyone unfamiliar with the context for the strange reasoning behind this so it doesn't look like an error, and also so it is clear when this can be cleaned up someday once the input system is more feature-complete.